redbot/cogs/audio/converters.py

```diff
@@ -176,31 +162,15 @@ class ScopeParser(commands.Converter):
             target_user = None
             user_raw = " ".join(author).strip()
             if user_raw.isnumeric():
-                user_raw = int(user_raw)
-                try:
-                    target_user = ctx.bot.get_user(user_raw)
-                except Exception:
-                    target_user = None
-                user_raw = str(user_raw)
+                target_user = ctx.bot.get_user(int(user_raw))
```

Comment:
Running a command such as `[p]playlist start foo --author ½` when a playlist named `foo` exist causes a conversion error due to the fact that `½` is numeric but cannot be `int`ed.

Reply:
#3089 should  fix this